spotui/src/PlaylistMenu.py

```diff
@@ -39,9 +46,27 @@ def __select_playlist(self, playlist_name, playlist_id, playlist_uri):
         self.change_tracklist(self.api.get_playlist_tracks(playlist_id),
                               playlist_name, playlist_uri)
 
-    def __map_playlists(self, item):
-        available_space = self.endx - self.startx - 6
+    def __map_playlists(self, item, current_playlist_uri):
+        available_space = self.endx - self.startx - 3
+        
         item["text"] = truncate(item["text"], available_space)
+        
+
+        #injection of current playlist
+        #TODO: update this list when a new playlist is playing
+        #TODO: check context on new song and update panels as necessary
+        if str(item["uri"]) == current_playlist_uri:
+           
+            if len(item["text"].encode("utf-8")) < available_space:
+                item["text"] = pad_str(item["text"], available_space - 2)
+
+            item["text"] = "{0} ".format(item["text"])
+        else:    
+            #TODO: ljust does not account for bytes
+            item["text"] = pad_str(item["text"], available_space)
+        #end injection
```

Comment:
`implement a public method called refresh_now_playing` sm:o:rt. will definitely do it that way.

I also went through most of utf-8 characters and found replacement icons for these, since these use nerdfont and aren't checking if that setting is turned on or not.